packages/test/test-end-to-end-tests/src/test/summarizeRestart.spec.ts

```diff
@@ -188,7 +188,7 @@ describeNoCompat("Summarizer closes instead of refreshing", (getTestObjectProvid
 			{
 				eventName: "fluid:telemetry:Summarizer:Running:Summarize_cancel",
 				category: "generic",
-				error: "disconnected",
+				error: "summary state stale",
```

Comment:
What's the plan for `trySummarizeWithStaticAttempts`, which is using the deprecated option `refreshLatestAck`? Once we roll out Navin's thing maybe we'll remove that too, but until then it seems like it needs to stay?

Reply:
Not sure, I don't know what that plan is